search-service/src/test/resources/ngsild/beehive_with_empty_temporal_attribute_evolution.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "id" : "urn:ngsi-ld:BeeHive:TESTC",
+  "type" : "BeeHive",
+  "incoming" : { },
```

Comment:
if you apply the same logic than for temporalValues, I would expect `incoming: []` instead